package.json

```diff
@@ -31,7 +31,7 @@
     "eslint-plugin-react-hooks": "^4.1.2"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "4.12.0",
+    "@typescript-eslint/eslint-plugin": "4.14.1",
```

Comment:
> __[Goodcheck]__ Don't forget updating `peerDependencies` when updating `devDependencies`. ([view](https://sider.review/gh/repos/72121459/pulls/864#issue-5354327))
>
> <details>
>
> | Rule |
> | ---- |
> | `update-peer-dependencies` |
>
> You can [close this issue](https://sider.review/gh/repos/72121459/pulls/864#issue-5354327) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue, because:
- When not updating `peerDependencies`.